BeeStation/BeeStation-Hornet
# Xenomorphs Stop Gasping due to atmospheric conditions

## About The Pull Request
Fixes Xenomorphs, which don't exactly breathe air, from gasping in low oxy or vacuumed environments.

Good for mobs who can make use of breathing some specific gases, but do not need to breathe to survive(IE: Xenomorphs).


## Why It's Good For The Game
closes #5826 
Xenos no longer gasp despite not needing O2 or otherwise. Still allows xenos to breathe plasma. No more being outed during ventcrawls or other situations where you may be in low O2 or vacuumed environments.


## Testing Procedure

1. Ghost yourself roundstart
2. ensure you have xenomorphs enabled in preferences
3. trigger-event
4. select alien infestion
5. click yes on the ghost role prompt
6. once spawned, ventcrawl into atmos
7. navigate towards the mixing output pipenet(the blank grey pipes)
8. Vibe for a few minutes
9. No Gasp? Working properly.

## TODO

- [x] What Ike Said

## Changelog
:cl: DatBoiTim
fix: Xenomorphs no longer gasp in low oxy/vaccuumed environments
/:cl:
